docs/static_site/src/pages/api/api.html

```diff
@@ -123,26 +74,11 @@ <h4>Python-first API</h4>
 {%- endfor -%}
 <h2>Other Bindings</h2>
 <div class="row">
-{%- for doc in page.docs -%}
-  {%- if doc.tag != 'python' -%}
-  <div class="col-4">
-      <div class="docs-card">
-          <div class="docs-logo-container">
-              <img class="docs-logo-image" src="{{doc.icon | relative_url}}">
-          </div>
-          <div class="docs-action-btn">
-            <a href="{{doc.guide_link | relative_url}}"> <img src="{{'assets/img/compass.svg' | relative_url}}" class="docs-logo-docs">{{doc.title}} Guide  <span class="span-accented">›</span></a>
-          </div>
-          <div class="docs-action-btn">
-            <a href="{{doc.tutorial_link | relative_url}}"> <img src="{{'assets/img/video-tutorial.svg' | relative_url}}" class="docs-logo-docs">{{doc.title}} Tutorials  <span class="span-accented">›</span></a>
-          </div>
-          <div class="docs-action-btn">
-            <a href="{{doc.api_link | relative_url}}"> <img src="{{'assets/img/api.svg' | relative_url}}" class="docs-logo-docs">{{doc.title}} API Reference  <span class="span-accented">›</span></a>
-          </div>
-      </div>
+  <div class="language-binding-banner">
+    <h4>Call for Contribution</h4>
+    The language bindings of MXNet 1.x were deleted in 2.x due to some C APIs being deprecated and the bindings did rely on the deprecated APIs.
+    Given that MXNet has C APIs and other language bindings can be added. Your contribution is welcome.
```

Comment:
```suggestion
    MXNet's new C APIs in v2.x can be used to reestablish your preferred language binding. Your contribution is welcome!
```
Can you double check that folks should be using the CPP API instead? Previously this was always a debate since the C API was supposed to be internal only.